docs/validators/emergency-upgrades.md

```diff
@@ -0,0 +1,83 @@
+---
+order: 5
+title: Emergency Upgrades
+---
+
+## **Introduction**
```

Comment:
might also want to provide quick definitions in here since you mention "coordinated" and "un-coordinated" in the sections below (line 22)

ie, 

state-breaking: requires a chain halt before upgrading 
coordinated upgrade: all validators must set a specific halt-height and run the new binary at that time.
un-coordinated upgrade: patch can be applied at the validators soonest availability